src/components/GrowlNotification/index.tsx

```diff
@@ -16,15 +19,26 @@ const INACTIVE_POSITION_Y = -255;
 
 const PressableWithoutFeedback = Pressables.PressableWithoutFeedback;
 
-function GrowlNotification(_, ref) {
+function GrowlNotification(_: unknown, ref: ForwardedRef<GrowlRef>) {
     const translateY = useRef(new Animated.Value(INACTIVE_POSITION_Y)).current;
     const [bodyText, setBodyText] = useState('');
     const [type, setType] = useState('success');
-    const [duration, setDuration] = useState();
+    const [duration, setDuration] = useState<number | undefined>();
```

Comment:
```suggestion
    const [duration, setDuration] = useState<number>();
```

You don't have to explicitly specify `undefined` if you leave the initial value of the state (i.e. the argument to `useState`) empty.